### PR TITLE
Tickets/DM-3496

### PR DESCRIPTION
**Pull request by jbecla**

### CODE REVIEW

File: python/lsst/dax/imgserv/imageREST_v0.py [hunk 1]
```diff
@@ -196,10 +208,11 @@ def getISkyMapDeepCoaddCutoutPixel():
     '''
     return _getISkyMapDeepCoaddCutout(request, 'pixel')
 
-def _getISkyMapDeepCoaddCutout(request, units):
+def _getISkyMapDeepCoaddCutout(request, units, source="/lsst7/releaseW13EP"):
```
Comment (timj) [inline python/lsst/dax/imgserv/imageREST_v0.py]: should this default be read from a config file?

Comment (jbecla) [inline python/lsst/dax/imgserv/imageREST_v0.py]: it will be passed as part of URI (very soon), see DM-3571

Comment (timj) [inline python/lsst/dax/imgserv/imageREST_v0.py]: and then the default will be removed?

Comment (jbecla) [inline python/lsst/dax/imgserv/imageREST_v0.py]: yes we are not attached to that default in any way :)

**Pull request merged**